<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="win764bit" name="persistent.MavenFirstPrj.LoginFunctionTestClass" tests="3" failures="1" timestamp="4 Aug 2014 22:30:01 GMT" time="0.008" errors="0">
  <testcase name="VerifyInvalidLoginl" time="0.001" classname="persistent.MavenFirstPrj.LoginFunctionTestClass"/>
  <testcase name="VerifyForgotPassword" time="0.006" classname="persistent.MavenFirstPrj.LoginFunctionTestClass"/>
  <testcase name="VerifyLoginSucessful" time="0.001" classname="persistent.MavenFirstPrj.LoginFunctionTestClass">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at persistent.MavenFirstPrj.Login.getWordpressURL(Login.java:23)
	at persistent.MavenFirstPrj.Login.Login(Login.java:47)
	at persistent.MavenFirstPrj.LoginFunctionTestClass.VerifyLoginSucessful(LoginFunctionTestClass.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- VerifyLoginSucessful -->
</testsuite> <!-- persistent.MavenFirstPrj.LoginFunctionTestClass -->
